.azure-pipelines/util/test-steps.yml

```diff
@@ -7,18 +7,18 @@ parameters:
 steps:
 - template: download-build-steps.yml
   parameters:
-    osName: ${{ parameters.osName }}
+    artifactName: build-${{ parameters.testFramework }}
```

Comment:
My consideration is we will support test on .NET framework soon. If so, it will generate 2 sets of artifacts. netcoreapp3.1 is used by PowerShell 7 on Windows, Linux, and MacOs. net48 will be used by Windows PowerShell. Does it make sense?